watchmal/dataset/cnn_mpmt/cnn_mpmt_dataset.py

```diff
@@ -71,24 +78,116 @@ def  __getitem__(self, item):
         data_dict = super().__getitem__(item)
 
         processed_data = from_numpy(self.process_data(self.event_hit_pmts, self.event_hit_charges))
+        
         processed_data = du.apply_random_transformations(self.transforms, processed_data)
-
+        
+        # Add padding
+        if self.pad:
+            processed_data = transformations.mpmtPadding(processed_data, self.barrel_rows)
```

Comment:
It looks like it's still calling `mpmtPadding` from `transformations`. I think this should now be `self.mpmtPadding`